src/services/timer.js

```diff
@@ -33,7 +33,7 @@ export const decrementOneSec = (mins, secs, timeDecrementedCallback, timeOverCal
  * @return {object} next period
  */
 export const determineNextPeriod = (currentPeriod, counter) => {
-  if (counter === 7) {
+  if (counter === 6) {
```

Comment:
Why are you changing this?

Reply:
it is was not work with ``handleNextClickso`` I think if I change it maybe it was work with ``handleNextClick``. so now I realize this change it was not necessary.  I had Restore it to previous situation.